app/view/grid/GridFiltersExample.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Example classes for a CpsiMapview.view.grid.Grid grid
+ */
+
+
+var storeConfig = {
+    type: 'array',
+    fields: ['name'],
+    data: [
+        ['Carlow'],
+        ['Cavan'],
+        ['Clare'],
+        ['Cork City'],
+        ['Cork County'],
+        ['DLRCC'],
+        ['Donegal County Council'],
+        ['Dublin City'],
+        ['Fingal'],
+        ['Galway City'],
+        ['Galway County'],
+        ['Kerry'],
+        ['Kildare'],
+        ['Kilkenny CC'],
+        ['Laois'],
+        ['Leitrim'],
+        ['Limerick'],
+        ['Longford'],
+        ['Louth'],
+        ['Mayo'],
+        ['Meath'],
+        ['Monaghan'],
+        ['Offaly'],
+        ['Roscommon CC'],
+        ['Sligo'],
+        ['South Dublin Co Co'],
+        ['Tipperary'],
+        ['Waterford'],
+        ['Westmeath'],
+        ['Wexford'],
+        ['Wicklow']
+    ]
+};
+
+
+// define the model
+Ext.define('CpsiMapview.model.GridFiltersExample', {
+    requires: 'GeoExt.data.model.Feature',
+    extend: 'GeoExt.data.model.Feature',
+    idProperty: 'OBJECTID',
+    fields: [
+        {
+            name: 'Unique_ID',
+            type: 'string'
+        },
+        {
+            name: 'GREENFIELD',
+            type: 'string'
+        },
+        {
+            name: 'AUTHORITY',
+            type: 'string'
+        },
+        {
+            name: 'Location',
+            type: 'string'
+        },
+        {
+            name: 'Calc_Area',
+            type: 'float'
+        }
+    ]
+});
+
+// create the store
+Ext.define('CpsiMapview.store.GridFiltersExample', {
+    extend: 'CpsiMapview.store.WfsFeatures',
+    alias: 'store.GridFiltersExample',
+    //url: 'https://w08-mapserver.compass.ie/mapserver/?map=/MapServer/apps/mapview-demo/example.map&',
```

Comment:
@geographika is there any reason why you are pointing to localhost?
for me it only works for the other URL.

Reply:
Sorry I forgot to revert in the checkin!

Reply:
alright, I will fix it in my PR regarding the list filter